libcodechecker/server/client_db_access_handler.py

```diff
@@ -26,6 +26,7 @@
 from sqlalchemy import func
 
 import shared
+from shared import constants as shared_constants
```

Comment:
`shared.constants` could be used if only `shared` is imported its length is the same as `shared_constants`